ipatests/test_integration/test_acme.py

```diff
@@ -583,6 +583,9 @@ def issue_and_expire_cert(self):
             tasks.kdestroy_all(host)
             tasks.move_date(host, 'stop', '+90days')
 
+        # wait for sssd to cache user after date move
+        time.sleep(10)
+
```

Comment:
Can you expand on this? What user does sssd need to cache? Can sss_cache be used instead?